src/system/AdminModule/Controller/AdminInterfaceController.php

```diff
@@ -0,0 +1,469 @@
+<?php
+/**
+ * Copyright Zikula Foundation 2015 - Zikula Application Framework
+ *
+ * This work is contributed to the Zikula Foundation under one or more
+ * Contributor Agreements and licensed to You under the following license:
+ *
+ * @license GNU/LGPLv3 (or at your option, any later version).
+ *
+ * Please see the NOTICE file distributed with this source code for further
+ * information regarding copyright and licensing.
+ */
+namespace Zikula\AdminModule\Controller;
+
+use Symfony\Component\Security\Core\Exception\AccessDeniedException;
+use Symfony\Component\HttpFoundation\Response;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route; // used in annotations - do not remove
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method; // used in annotations - do not remove
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Zikula\Core\Controller\AbstractController;
+use Zikula\Bundle\CoreBundle\Console\Application;
+
+/**
+ * @Route("/admininterface")
+ */
+class AdminInterfaceController extends AbstractController
+{
+
+    /**
+     * @Route("/header")
+     *
+     * Open the admin container
+     *
+     * @return Response symfony response object
+     */
+    public function headerAction()
+    {
+        $masterRequest = $this->get('request_stack')->getMasterRequest();
+        $caller = [];
+        $caller['_zkModule'] = $masterRequest->attributes->get('_zkModule');
+        $caller['_zkType'] = $masterRequest->attributes->get('_zkType');
+        $caller['_zkFunc'] = $masterRequest->attributes->get('_zkFunc');
+        return $this->render("ZikulaAdminModule:AdminInterface:header.html.twig", [
+            'caller' => $caller
+        ]);
+    }
+
+    /**
+     * @Route("/footer")
+     *
+     * Close the admin container
+     *
+     * @return Response symfony response object
+     */
+    public function footerAction()
+    {
+        $masterRequest = $this->get('request_stack')->getMasterRequest();
+        $caller = [];
+        $caller['_zkModule'] = $masterRequest->attributes->get('_zkModule');
+        $caller['info'] = \ModUtil::getInfoFromName($caller['_zkModule']);
```

Comment:
what kind of `info` is needed here? I think I remember only the module's version? If so, only pass the required information. Also let's use a non-legacy method like:

```php
$moduleEntity = $this->getDoctrine()->getManager()->getRepository('ZikulaExtensionsModule:ExtensionEntity')->findOneBy(['name' => $caller['_zkModule']);
$caller['version'] = $moduleEntity->getVersion();
```